oeps/oep-38-Data-Modeling.rst

```diff
@@ -0,0 +1,442 @@
+.. vim: syntax=rst
+
+===============================
+
+OEP-38:
+
+===============================
+
++---------------+------------------------------------------------------------+
+
+| OEP | :doc:` |
+
++---------------+------------------------------------------------------------+
+
+| Title | Data Modeling Best Practices |
+
++---------------+------------------------------------------------------------+
+
+| Last Modified | 2019-08-01 |
+
++---------------+------------------------------------------------------------+
+
+| Author | Brian Beggs <bbeggs@edx.org> |
+
++---------------+------------------------------------------------------------+
+
+| Arbiter | Albemarle |
+
++---------------+------------------------------------------------------------+
+
+| Status | Provisional |
+
++---------------+------------------------------------------------------------+
+
+| Type | Architecture |
+
++---------------+------------------------------------------------------------+
+
+| Created | 2019-05-01 |
+
++---------------+------------------------------------------------------------+
+
+| Review Period | 2019-08-01 - 2019/08/16 |
+
++---------------+------------------------------------------------------------+
+
+| Resolution | - 'Original pull request' |
+
+| | - 'Update pull request' |
+
++---------------+------------------------------------------------------------+
+
+| References | - 'OEP-002' |
+
++---------------+------------------------------------------------------------+
+
+
+.. \_Original pull request:
+
+.. \_Update pull request:
+
+.. \_OEP-002:
+https://open-edx-proposals.readthedocs.io/en/latest/oep-0002-bp-repo-metadata.html
+
+Abstract
+
+========
+
+In order to obtain the highest possible value from data collected in the
+edX ecosystem, this document attempts to provide guiding thoughts and
+principles on data modeling.
+
+These data modeling guidelines are broken down into three sections:
+
+-  Guiding Principles
+
+-  Data Modeling Standards
+
+-  Best Practices
+
+Guiding Principles - This section is intended to give the reader a
+framework to think about data modeling at edX. While no specific
+instructions are given in this section, it should act as a framework to
+help the reader think about data modeling on the whole at edX.
```

Comment:
This seems to repeat itself and "no specific instructions" seems to contradict what we're doing with "Data modeling standards".

Reply:
Reworded this.